packages/token-sdk/src/metadata/client/metaplex-client.ts

```diff
@@ -4,6 +4,9 @@ import invariant from "tiny-invariant";
 
 const METADATA_PROGRAM_ID = new PublicKey("metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s");
 
+// Metadata should be a just tiny JSON file, 5000ms should be sufficient for most cases
+const DEFAULT_GET_OFF_CHAIN_METADATA_TIMEOUT_MS = 5000;
```

Comment:
I think we could consider making this timeout even smaller - maybe 1s or 2s?

Reply:
Thanks for the review.
yep, updated to 2s 👍